### ISSUE TITLE
PosRep: remove currency columns and display currencies next to numbers

### ISSUE THREAD

**Issue by Eddman** [Deutsche-Boerse-Risk/DAVe-UI#22]
![image](https://user-images.githubusercontent.com/209500/27374025-a4ebfaf2-566a-11e7-83a9-5a1a4862274b.png)

**Comment by markadir** [comment]
I would only remove the Clearing currency column and Keep the product currency, when you are in the details. What are the currencies of Vega, Rho etc - I asusme they are not FX adjusted? I also Assume EuroDelta is FX adjusted?

The column "Flex Contract Symbol" should be removed. 

Rename in the columns "Liqui" to either "Liquidation" or "Liq.". 

There are no smaller "net positions" then 1 hence comma values are not needed.

**Comment by Eddman** [comment]
@markadir Point one is the same as in #24 right?

` There are no smaller "net positions" then 1 hence comma values are not needed.`  is related to #32.

`What are the currencies of Vega, Rho etc - I asusme they are not FX adjusted? I also Assume EuroDelta is FX adjusted?` @zerola, @halajos?

**Comment by Eddman** [comment]
@orsamir Could you please answer the question:
`What are the currencies of Vega, Rho etc - I asusme they are not FX adjusted? I also Assume EuroDelta is FX adjusted?`

**Comment by orsamir** [comment]
normalized greeks are all converted to EUR

**Comment by markadir** [comment]
@PavlaSip : Please provide the currency (Clearing Cur, POrod Cur) for each column.

**Comment by PavlaSip** [comment]
So based on specs I found the following: 

- Position VaR: Clearing cur
- LA: Clearing cur
- CorrBreak: Clearing cur
- CompError: Clearing cur
- LongOptCredit: Clearing cur
- PremPay: Product cur
- PremMgn: Clearing cur
- Gamma, Vega, Rho, Theta: all in **Product cur**
- mVar: Clearing cur

@iyigmel: in the CI720 the greeks are shown twice - once at Position Level in product currency (S4) and secondly they are also converted to EUR which is then shown in M1 template but these metrics include the "Euro" in the name (e.g. "Vega in Euro")

**Comment by iyigmel** [comment]
regarding greeks, this is exactly how I expect....However, could you please check premMgn again? on position level it should be also product currency . 

•Position VaR: Clearing cur
•LA: Clearing cur
•CorrBreak: Clearing cur
•CompError: Clearing cur
•LongOptCredit: Clearing cur
•mVar: Clearing cur

and 

•PremPay: Product cur
•PremMgn: Product cur
•Gamma, Vega, Rho, Theta: all in Product cur

**Comment by PavlaSip** [comment]
@iyigmel: yes, you are correct, it is firstly calculated in product currency and only afterwards the positions' PremMgn are coverted to Clearing cur for the aggregation purposes. It wasn't clear to me if we take the originally calculated PremMgn or the one used later for aggregation.  
@Eddman I have corrected the list in my comment above.

**Comment by orsamir** [comment]
@PavlaSip: In the data Prisma sends to Dave the greeks are normalized using underlying price and **converted to EUR.**

**Comment by orsamir** [comment]
@iyigmel @PavlaSip on Position Level Prisma provides the premium margin **in product currency.**

**Comment by Eddman** [comment]
@orsamir:
```
In the data Prisma sends to Dave the greeks are normalized using underlying price and converted to EUR.
```
EUR or Clearing currency? What if user has selected clearing currency CHF? Is it still EUR?

**Comment by orsamir** [comment]
EUR

**Comment by markadir** [comment]
I think it still makes sense to keep it in EUR since our main products are in EUR. It doesn't really makes sense to convert EuroStoxx to CHF Delta. With column names EUR Delta it is clear to the user that Delta is normalized to EUR. We should do it with all EUR greeks and Change column names to say which currency is used to normalize. Hence Vega should be changed to EUR Vega, Gamma Should be changed to EUR Gamma. Same with Rho and Theta.

**Comment by markadir** [comment]
For all other columns the currency should be displayed next to the number. 
•Position VaR: Clearing cur
•LA: Clearing cur
•CorrBreak: Clearing cur
•CompError: Clearing cur
•LongOptCredit: Clearing cur
•mVar: Clearing cur
•PremPay: Product cur
•PremMgn: Product cur

**Comment by Eddman** [comment]
@markadir @PavlaSip @iyigmel Please verify and close the issue.

**Comment by PavlaSip** [comment]
To me it looks good.

**Comment by Eddman** [comment]
@markadir ?

**Comment by markadir** [comment]
I am missing the Variation Margin column. If we have PremPay we should also have VM...

**Comment by orsamir** [comment]
VM+Prem payment is provided as one number by Prisma (PositionReport.variation_premium_payment). The column name should somehow indicate that.

**Comment by markadir** [comment]
Yep, thats best if it's one column but then we should rename the column to "VM and Premium Payment"

**Comment by PavlaSip** [comment]
Just a thought: on the other views this column is called "Variation Premium Payment" - maybe better to use the same to remain consistent?
